.github/workflows/automation-scripts/update-changelogs.mjs

```diff
@@ -63,13 +69,12 @@ await $`rush publish --regenerate-changelogs`;
 // await $`git checkout -b finalize-release-X.X.X`;
 // targetBranch = "finalize-release-X.X.X"
 /*********************************************************************/
-targetBranch = targetBranch.replace("origin/", "");
 await $`git add .`;
 await $`git commit -m "${commitMessage} Changelogs"`;
 await $`rush change --bulk --message "" --bump-type none`;
 await $`git add .`;
 await $`git commit --amend --no-edit`;
-await $`git push HEAD:${targetBranch}`;
+await $`git push origin HEAD:${targetBranch}`;
```

Comment:
do you need this `HEAD:` syntax?

Reply:
i don't think so, I think its usually if your local branch name isn't the same as the upstream branch.
But because the branch is changing during the workflow i thought stating the branch would be best, and I thought this was the most appropriate way to write it.
git push origin HEAD:master
    Push the current branch to the remote ref matching master in the origin repository. This form is convenient to push the current branch without thinking about its local name.

Pretty sure it could be written a few other ways, but i thought this would be the most explicit